src/doc/reference.md

```diff
@@ -1983,6 +1983,7 @@ macro scope.
 
 ### Miscellaneous attributes
 
+- `deprecated` - Allows using the `#[deprecated]` attribute.
```

Comment:
@durka, ah, I see. I just moved the existing reference to `deprecated`; I missed seeing that the verbiage needed to be changed as well. As the structure of the document currently stands, this is the list of attributes which aren't specifically scoped to crates, modules, functions, items, etc.

It looks like the wording should presumably be altered to say something like this:

> `deprecated` - Marks the item as deprecated.

Presumably that should be expanded a bit, too; what's the actual syntax for the use of the deprecated attribute—version number and comment? I can't find that attribute itself documented anywhere. If someone can give me the relevant info, I'll fill it out with that.

(Which, of course, is exactly why we need https://github.com/rust-lang/rfcs/pull/1636. :wink:)